diff -ruN orig_jaseci_ai_kit/modules/cl_summer/cl_summer.py jaseci_ai_kit/modules/cl_summer/cl_summer.py
--- orig_jaseci_ai_kit/modules/cl_summer/cl_summer.py	2022-11-17 16:16:40.916150000 -0500
+++ jaseci_ai_kit/modules/cl_summer/cl_summer.py	2022-11-17 16:16:33.526150000 -0500
@@ -10,8 +10,8 @@
 from sumy.utils import get_stop_words
 import nltk

-nltk.download("punkt")
-nltk.download("stopwords")
+# nltk.download("punkt")
+# nltk.download("stopwords")

 # summarizes the text / url passed to the endpoint and
 # returns summary based on the sentences.
diff -ruN orig_jaseci_ai_kit/modules/text_seg/text_seg.py jaseci_ai_kit/modules/text_seg/text_seg.py
--- orig_jaseci_ai_kit/modules/text_seg/text_seg.py	2022-11-17 16:16:40.926150000 -0500
+++ jaseci_ai_kit/modules/text_seg/text_seg.py	2022-11-17 16:16:33.556150000 -0500
@@ -5,12 +5,14 @@
 from jaseci.actions.live_actions import jaseci_action

 # loading segmentation model from hugging face
-tokenizer = AutoTokenizer.from_pretrained("dennlinger/roberta-cls-consec")
+tokenizer = AutoTokenizer.from_pretrained(
+    "/trained_models/roberta-cls-consec", local_files_only=True
+)
 model = AutoModelForSequenceClassification.from_pretrained(
-    "dennlinger/roberta-cls-consec"
+    "/trained_models/roberta-cls-consec", local_files_only=True
 )
 # Download the pretrained model pipeline
-spacy.cli.download("en_core_web_sm")
+# spacy.cli.download("en_core_web_sm")
 # loading space model for sentence tokenization
 pipeline = spacy.load("en_core_web_sm")

diff -ruN orig_jaseci_ai_kit/modules/use_enc/use_enc.py jaseci_ai_kit/modules/use_enc/use_enc.py
--- orig_jaseci_ai_kit/modules/use_enc/use_enc.py	2022-11-17 16:16:40.926150000 -0500
+++ jaseci_ai_kit/modules/use_enc/use_enc.py	2022-11-17 16:16:33.566150000 -0500
@@ -6,7 +6,7 @@
 from typing import Union


-module = hub.load("https://tfhub.dev/google/universal-sentence-encoder/4")
+module = hub.load("/trained_models/universal-sentence-encoder_4/")


 @jaseci_action(act_group=["use"], aliases=["get_embedding"], allow_remote=True)
diff -ruN orig_jaseci_ai_kit/modules/use_qa/use_qa.py jaseci_ai_kit/modules/use_qa/use_qa.py
--- orig_jaseci_ai_kit/modules/use_qa/use_qa.py	2022-11-17 16:16:40.926150000 -0500
+++ jaseci_ai_kit/modules/use_qa/use_qa.py	2022-11-17 16:16:33.566150000 -0500
@@ -6,9 +6,7 @@
 from typing import Union


-module = hub.load(
-    "https://tfhub.dev/google/universal-sentence-encoder-multilingual-qa/3"
-)
+module = hub.load("/trained_models/universal-sentence-encoder-multilingual-qa_3")


 @jaseci_action(act_group=["use"], aliases=["enc_question"], allow_remote=True)
